src/activities/assetProcessor/APAttachedProcessorsCollectionEntity.js

```diff
@@ -0,0 +1,94 @@
+import { Classes } from '../../hypermedia-constants.js';
+import { Entity } from '../../es6/Entity.js';
+
+class APAttachedProcessorEntity extends Entity {
+
+	/**
+	 * @returns {string} The external deployment id of the deployment this processor belongs to.
+	 */
+	externalDeploymentId() {
+		return this._entity && this._entity.properties && this._entity.properties.externalDeploymentId;
```

Comment:
This is interesting. I see it used in other places and reading it would expect it to return true/false but I'm assuming it sort of front-loads true/false checks and then returns the actual value kinda?

really weird